### ISSUE TITLE
Keboard input ignored after 'autoconfig' is populated by online update

### ISSUE THREAD

**Issue by sylverpyro** [libretro/RetroArch#2061]
I'm running into an issue where, on a brand new install of retroarch stable (1.2.2) on linux, keyboard control of the xmb menu works just fine until I download the autoconfig profiles (Online Updater -> Update Autoconfig Profiles") and restart retroarch. At this point, the Xbox controllers I have plugged in get detected and work properly, but all keyboard input gets ignored (even if I restart with no controllers plugged in). If I remove the autoconfig directory (or rename it) and restart, keyboard input is recognized again.

**Comment by fr500** [comment]
Can you state your platform at least?

**Comment by sylverpyro** [comment]
Arch Linux.  Kernel 4.1.6-1-ARCH.  The keyboard is a wireless USB keyboard with the identifier: ID 045e:0800 Microsoft Corp.

**Comment by fr500** [comment]
Interesing, I guess the matching system is recognizing your keyboard as a
gamepad because of the name, I'll check the autoconf files, maybe there is
something that can be done

**Comment by fr500** [comment]
Can you recompile, then run with verbose logging (retroarch --verbose --log-file retroarch.log --menu)
And paste the log somewhere, I improved the logging for the autoconf function so we can try to debug your issue.

**Comment by sylverpyro** [comment]
So, compiling straight out of git (rather than using stable 1.2.2) and running that, the issue vanishes.  It must have been already fixed in some other post 1.2.2 commit.  

I pulled logs from each version in case they pointed to something obvious that changed, but the two look fairly similar: 

]$ grep "udev\|Microsoft" ./retroarch*.log | grep -v "configuration file:"
./retroarch-git.log:RetroArch [INFO] :: [udev]: Plugged pad: Microsoft Microsoft® Nano Transceiver v2.0 (045e:0800) on port #0.
./retroarch-git.log:RetroArch [INFO] :: Autodetect: no profiles found for Microsoft Microsoft® Nano Transceiver v2.0 (1118/2048)RetroArch [INFO] :: [udev]: Pad #0 (/dev/input/event3) supports 0 force feedback effects.
./retroarch-git.log:RetroArch [INFO] :: Found joypad driver: "udev".
./retroarch.log:RetroArch [INFO] :: [udev]: Plugged pad: Microsoft Microsoft® Nano Transceiver v2.0 (045e:0800) on port #0.
./retroarch.log:RetroArch [INFO] :: Microsoft Microsoft® Nano Transceiver v2.0 configured in port #0.
./retroarch.log:RetroArch [INFO] :: [udev]: Pad #0 (/dev/input/event3) supports 0 force feedback effects.
./retroarch.log:RetroArch [INFO] :: Found joypad driver: "udev".

Both identify the keyboard transceiver and assign it to port 0, but only the -git version then actually responds to the input.  

Either way, it's resolved in git, so I guess this can be marked resolved.  

If you want the full log files still for any reason, just let me know.
